easybuild/easyconfigs/i/igraph/igraph-0.7.1-intel-2016b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'igraph'
+version = '0.7.1'
+
+homepage = 'http://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on 
+efficiency, portability and ease of use. igraph is open source and free. igraph can be 
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+sources = ['igraph-%(version)s.tar.gz']
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+
+builddependencies = [('Autotools', '20150215')]
+
+patches = [
+	# Author: Maxime Boissonneault <maxime.boissonneault@calculquebec.ca>
+	# Remove explicit linking on -llapack and -llblas, which are not found when linking against MKL
```

Comment:
@mboisson now that we have the patch description in the patches themselves, we can clean this up to just:

```python
patches = [
    'igraph-%(version)s-no-lapack-no-blas.patch',
    'igraph-%(version)s-fix-aclocal-version.patch',
]